code/modules/antagonists/bloodsuckers/powers/olfaction.dm

```diff
@@ -137,6 +139,8 @@
 		Selecting one of those scents will grant you a new ability to follow that scent. This will create a visible trail to follow to the owner of that scent.\n\
 		WARNING: it will be difficult to see around you while following a scent trail due to the the color being drained from all your vision except for the trail and your target."
 
+	cooldown = 60 SECONDS
```

Comment:
why 60 secondes and not 1 minutes

Reply:
it is likely to be moved around in seconds rather than minutes in future balancing, for example `1.1 MINUTES` is unreadable